### ISSUE TITLE
Is there a reason why integer models are not supported

### ISSUE THREAD

**Issue by peno64** [Gurobi/gurobi-modelanalyzer#4]
Can the tool not just ignore integer constraints, ie use the relaxed model?

**Comment by mattmilten** [comment]
No, there is no specific reason to not perform the linear relaxation internally. We will certainly consider implementing that as a convenience feature.
